test/server/runner-test.js

```
@@ -138,8 +138,73 @@ describe('Runner', function () {
                 .catch(done);
         });
 
+        it('Should raise error if browser disconnected before bootstrapping', function (done) {
```

Should raise an error if the browser disconnects before bootstrapping